apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -322,6 +330,19 @@ const styles = {
     flexDirection: 'row',
     marginTop: 40
   },
+  customText: {
+    marginRight: 2,
```

Comment:
Fun fact! There's a few shortened syntaxes you can use for this:
margin: <top/bottom> <left/right> => `margin: 0 2`
margin: <top> <right> <bottom> <left> => `margin: 0 2 0 2`
No need to change anything here, just sharing another way to write it